village/src/components/NavBar.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import "./NavBar.css";
+
+const NavBar = () => {
+  return (
+    <nav>
+    <NavLink className="navigation" exact activeClassName="selected" to="/">
```

Comment:
Gray job using the NavLink component here and giving it am active name and adding the exact parameter here, well done.